drivers/tee/optee/supp.c

```diff
@@ -45,6 +45,7 @@ void optee_supp_uninit(struct optee_supp *supp)
 u32 optee_supp_thrd_req(struct tee_context *ctx, u32 func, size_t num_params,
 			struct tee_param *param)
 {
+	bool interruptable;
```

Comment:
Sorry to nitpick, but I don't think the spelling is correct. Should that be 'interrupted' instead? (past tense).